app/src/lib/text-token-parser.ts

```diff
@@ -130,39 +130,61 @@ export class Tokenizer {
     index: number,
     repository: GitHubRepository
   ): LookupResult | null {
-    let nextIndex = this.scanForEndOfWord(text, index)
-    let maybeIssue = text.slice(index, nextIndex)
-
-    // handle situation where issue reference is wrapped in parentheses
-    // like the generated "squash and merge" commits on GitHub
-    if (maybeIssue.endsWith(')')) {
-      nextIndex -= 1
-      maybeIssue = text.slice(index, nextIndex)
+    let nextIndex = index + 1
+    while (nextIndex < text.length && /\d/.test(text[nextIndex])) {
+      nextIndex += 1
+    }
+    if (nextIndex === index + 1) {
+      return null
     }
 
-    // release notes may add a full stop as part of formatting the entry
-    if (maybeIssue.endsWith('.')) {
-      nextIndex -= 1
-      maybeIssue = text.slice(index, nextIndex)
+    const maybeIssue = text.slice(index, nextIndex)
+
+    const id = parseInt(maybeIssue.substring(1), 10)
+    if (isNaN(id)) {
+      return null
     }
```

Comment:
@thinkyhead could you please explain these changes? 

It would look to me that now issues that end in a `.`, `)`, or `,` will now just return null?

Reply:
Also, thinking it would be nice to keep using the `scanForEndOfWord` since that has been tried and tested more many years.